grafana/scylla-dash-io-per-server.master.json

```diff
@@ -180,7 +180,7 @@
                         },
                         "targets": [
                             {
-                                "expr": "count(up{job=\"scylla\"})-count(round(rate(scylla_gossip_heart_beat{shard=\"0\"}[10s])))",
+                                "expr": "count(up{job=\"scylla\"})-sum(floor(rate(scylla_gossip_heart_beat{shard=\"0\"}[10s]) + 0.2))",
```

Comment:
@amnonh what does calc mean? why 0.2 ?
Would this value hold with different refresh or scrapping interval?